go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/jpillora/backoff v1.0.0
 	github.com/json-iterator/go v1.1.12
-	github.com/kopia/kopia v0.13.0-rc2
+	github.com/kopia/kopia v0.10.6
```

Comment:
👀 Should we add another filter? @julio-lopez

Reply:
yes, indeed